Test Project/Assets/Scripts/ToggleButtonColour.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ToggleButtonColour : MonoBehaviour {
+
+
+	#region Serialized Fields
+	
+	[SerializeField]
+	private List<Color> m_Colors;
+	[SerializeField] 
+
+	private Button m_TargetButton;
```

Comment:
Hmm, this still looks a little off. It should be:

```
[SerializeField]
private List<Color> m_Colors;

[SerializeField] 
private Button m_TargetButton;
```

Reply:
Well, that was just me being dumb... I clearly intended to put the blank line above SerializeField.